### PR TITLE
Adds a tip

### PR DESCRIPTION
**Pull request by Amelia0010**

### CODE REVIEW

**Comment by simonset55** [comment]
I'm a retard and don't understand this code change, please provide a description

**Comment by simonset55** [comment]
Don't mock me for my defects REEEEEEEEE

File: config/sillytips.txt [hunk 1]
```diff
@@ -23,3 +23,4 @@ Plenty of things that aren't traditionally considered weapons can still be used
 DEATH IS IMMINENT!
 This game is older than some of the people playing it.
 Do not go gentle into that good night.
+Meth can be ignited while inside people, causing it to blow up.
```
Comment (simonset55) [inline config/sillytips.txt]: Could change it to pyrotechnic chems instead of meth, since any chemical can be heated inside of people

Comment (Amelia0010) [reply config/sillytips.txt]: 🤔 config/sillytips.txt

>sillytips

Comment (simonset55) [reply config/sillytips.txt]: But it ain't really silly doe is it

**Review by ThatLing — APPROVED** [review]

File: config/sillytips.txt (hunk 1 above)
Comment (ThatLing) [reply config/sillytips.txt]: Move it to config/serioustips.txt

**Comment by Time-Green** [comment]
I think we had a problem with tips before, since you’ll need to manually change them on the host

**Comment by Amelia0010** [comment]
it's the config folder, so yeah

**Comment by Time-Green** [comment]
add:
You can't break the rules if there's no admins

File: config/sillytips.txt [hunk 2]
```diff
@@ -24,3 +24,4 @@ DEATH IS IMMINENT!
 This game is older than some of the people playing it.
 Do not go gentle into that good night.
 Meth can be ignited while inside people, causing it to blow up.
+You can't break the rules if there are no admins to enforce them.
```
Comment (ThatLing) [inline config/sillytips.txt]: Nuke this

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: config/sillytips.txt (hunk 2 above)
Comment (Amelia0010) [reply config/sillytips.txt]: 😕

**Review by ThatLing — APPROVED** [review]

**Comment by matskuman5** [comment]
Please no
retards will think that's real and grief

**Comment by Amelia0010** [comment]
image

**Pull request closed without merging**